modules/client-configuration/pages/client-proxy-script.adoc

```diff
@@ -0,0 +1,67 @@
+[[script-client-proxy]]
+= Register Clients  to a Proxy with a Bootstrap Script
+
+
+You can register clients through the {productname} Proxy with a bootstrap script.
+This is done almost the same way as registering clients directly with the {productname} Server.
+
+You create the bootstrap script in the {productname} Server container with the [command]``mgr-bootstrap`` command line tool.
+[command]``mgr-bootstrap`` requires some parameters such as the fully qualified domain name (FQDN) of the proxy, activation keys, or GPG keys.
+These parameters depend on your specific setup, but the FQDN of the proxy is mandatory.
+
+The bootstrap script then deploys all necessary information to the clients.
+
+
+
+.Procedure: Registering Clients to a Proxy with a Bootstrap Script
+
+. Create a client activation key on the {productname} Server using the {webui}.
+  For more information, see xref:client-configuration:activation-keys.adoc[].
+. On the container host of the server, start a terminial in the container:
+
++
+
+[source,shell]
+----
+mgrctl term
+----
+
++
+
+--
+.. In the container, to create the bootstrap script, execute the [command]``mgr-bootstrap`` command:
+
++
+
+[source,shell]
+----
+mgr-bootstrap --hostname=MLM_PROXY --activation-keys=ACTIVATION_KEY
+----
+
++
+
+Replace [literal]``MLM_PROXY`` with the host name of your proxy.
+   If needed, use the additional command line switches to tune your bootstrap script.
+   To view available options type [command]``mgr-bootstrap --help`` from the command line.
```

Comment:
I forgot to also mention that they should consider using custom name for the resulting script so all the bootstrap scripts are not mixed together.

E.g.:

Consider using [command]``--script bootstrap-MLM_PROXY.sh`` option to save bootstrap script with descriptive name.